soco/data_structures_entry.py

```diff
@@ -23,7 +23,7 @@
 _LOG.debug('%s imported', __name__)
 
 
-def from_didl_string(string):
+def from_didl_string(string, music_service_medata_hack=False):
```

Comment:
Is there a specific reason why you don't want to activate this by default?

Reply:
I preparing a new version of this, so I wanted to answer the feedback. Yes, at the time I didn't know that our music services would continue to be broken and I really wanted the music services to provide the metadata. But since we don't know if and when the music services will re-appear I will now do this per default.